skbeam/core/accumulators/binned_statistic.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, sample, statistic='mean',
             minimum and maximum values along each dimension.
 
         """
+        if mask is None:
+            mask = np.ones_like(sample)
```

Comment:
I have updated the documentation for mask specifying that 0-values are ignored.  Interesting that pyFAI does it the other way.  To add to what CJ wrote: our convention also allows img*mask to do something natural.  I'd be interested to hear what the pyFAI people have to say about the other convention.  Maybe we can ask them on our next phone call.